warehouse/accumulo-extensions/pom.xml

```diff
@@ -42,6 +42,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>${version.junit}</version>
```

Comment:
Do you need the version here?  Is it not inherited from the parent pom?

Reply:
IntelliJ flagged this instance of junit as not matching the parent pom

Reply:
is junit:junit in the dependency management section of the parent pom?  if not, it should be, and you shouldn't need the version here.

Reply:
I wonder if the test scope somehow mucks with that.  You may need to add the test scope for that in the management section as well.  Just a guess.

Reply:
he already updated it to remove the version.  it wasn't needed.

i would actually advise against explicitly listing the test scope when managing dependencies.  if you specify a scope, it will only manage it for that scope, and OTHER imported dependency management sections could override your compile time version.